watertap/unit_models/pressure_exchanger.py

```diff
@@ -320,26 +370,41 @@ def eq_equal_low_pressure(b, t):
                 == b.low_pressure_side.properties_in[t].pressure
             )
 
-        if self.config.has_mass_transfer:
+        if self.config.pressure_exchange_calculation is PressureExchangeType.efficiency:
 
-            @self.Constraint(
-                self.flowsheet().config.time,
-                self.config.property_package.phase_list,
-                self.config.property_package.component_list,
-                doc="Mass transfer from high pressure side",
-            )
-            def eq_mass_transfer_from_high_pressure_side(b, t, p, j):
-                comp = self.config.property_package.get_component(j)
-                return b.high_pressure_side.mass_transfer_term[
-                    t, p, j
-                ] == -b.mass_transfer_fraction_comp[
-                    t, j
-                ] * b.high_pressure_side.properties_in[
+            @self.Constraint(self.flowsheet().config.time, doc="Pressure transfer")
+            def eq_pressure_transfer(b, t):
+                return (
+                    b.low_pressure_side.deltaP[t]
+                    == b.efficiency_pressure_exchanger[t]
+                    * -b.high_pressure_side.deltaP[t]
+                )
+
+        elif (
+            self.config.pressure_exchange_calculation
+            is PressureExchangeType.high_pressure_difference
+        ):
+
+            @self.Constraint(self.flowsheet().config.time, doc="Pressure transfer")
+            def eq_pressure_transfer(b, t):
+                return b.high_pressure_side.properties_in[
```

Comment:
if you put parentheses around the statement, I think black won't make it look crazy:

`return (pressure_HPS_in = pressure_LPS_out + HPD)`

Reply:
I tried, but Black will directly reformat back to this formula and deleted the parentheses I added.